### PR TITLE
update layout page

### PR DESCRIPTION
**Pull request by szaimen**
Address part of https://github.com/nextcloud/documentation/issues/10392

TODO:
- [x] Update all screenshots
- [x] add links to penpot?
- [x] address reviews

### CODE REVIEW

File: developer_manual/images/talk-layout.png
Comment (szaimen) [inline developer_manual/images/talk-layout.png:1]: I guess I will need a better test account here in order to create a better screenshot. This is only a placeholder for now

File: developer_manual/design/layout.rst [hunk 1]
```diff
@@ -10,15 +10,16 @@ While deciding how you want your app to look, there are a number of factors to c
 * Responsiveness for different browsers, browser sizes and devices
 * Typical interface patterns in other similar apps in the market
 
-The `content Vue component <https://nextcloud-vue-components.netlify.app/#/Components/App%20containers?id=content>`_ wraps your entire app. While the arrangement of components in your app is dependent on what your app does, most Nextcloud apps typically have 3 levels of hierarchy. Some commonly used layouts are:
-
+The `content Vue component <https://nextcloud-vue-components.netlify.app/#/Components/App%20containers?id=ncappcontent>`_ wraps your entire app. While the arrangement of components in your app is dependent on what your app does, most Nextcloud apps typically have 3 levels of hierarchy. Some commonly used layouts are:
```
Comment (jancborchardt) [inline developer_manual/design/layout.rst:13]: For some reason the link to the overall section does not work properly, we have to link to the first subsection:
https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppContent?id=ncappcontent-1

File: developer_manual/design/layout.rst [hunk 2]
```diff
@@ -11,14 +11,15 @@ While deciding how you want your app to look, there are a number of factors to c
 * Typical interface patterns in other similar apps in the market
 
 The `content Vue component <https://nextcloud-vue-components.netlify.app/#/Components/App%20containers?id=ncappcontent>`_ wraps your entire app. While the arrangement of components in your app is dependent on what your app does, most Nextcloud apps typically have 3 levels of hierarchy. Some commonly used layouts are:
+`Penpot empty template https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=2783d7ad-98f2-804a-8002-750c2585d4f1&section=interactions&index=5&share-id=11fde340-21f4-802e-8002-8d8d305e7ab5`_
```
Comment (jancborchardt) [inline developer_manual/design/layout.rst:14]: Same here, best to use the same unified Penpot link as in https://github.com/nextcloud/documentation/pull/10431 :)

**Review by jancborchardt — CHANGES_REQUESTED** [review]
See comments on links, otherwise good!

File: developer_manual/design/layout.rst (hunk 2 above)
Comment (szaimen) [reply developer_manual/design/layout.rst:14]: This is the same link like in #10431 :)

File: developer_manual/design/layout.rst (hunk 1 above)
Comment (szaimen) [reply developer_manual/design/layout.rst]: fixed

**Comment by jancborchardt** [comment]
Small comment on the link `<>`, and I guess blocking still cause of the Mail screenshot here too?

File: developer_manual/design/layout.rst (hunk 2 above)
Comment (szaimen) [reply developer_manual/design/layout.rst]: true. not sure how that happened :D

**Review by marcoambrosini — APPROVED** [review]

File: developer_manual/images/mail-layout.png
[the remark this answers is not in the document]
Comment (szaimen) [reply developer_manual/images/mail-layout.png:1]: done

File: developer_manual/images/talk-layout.png
Comment (szaimen) [reply developer_manual/images/talk-layout.png:1]: done

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**